src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -546,8 +565,9 @@
                          (open-chat-context-menu message))}
       [react/view {:accessibility-label :chat-item}
        (let [incoming-group (and group-chat (not outgoing))]
-         [message-content message-body (merge message
-                                              {:current-public-key current-public-key
-                                               :group-chat         group-chat
-                                               :modal?             modal?
-                                               :incoming-group     incoming-group})])]]]))
+         [message-content (merge message
+                                 (when system-message? {:content-type constants/content-type-system-text})
```

Comment:
This is probably not the way to do it, if we want to have a `constansts/content-type-system-text` we should just generate this message with the right content-type (in status-go), so we avoid having to add it at runtime.